controllers/users.controller.js

```diff
@@ -35,15 +35,55 @@ exports.getUserGarments = async (userId, limit = 10, offset = 0) => {
 exports.addUserGarment = async (userId, requestBody) => {
     let foundUser = await User.findById(userId);
     foundUser.garments.push(requestBody);
-    let updatedUser = user.save();
+    let updatedUser = await foundUser.save();
 
     return {
         status: 200,
-        message: 'Updated user garment',
+        success: true,
         data: updatedUser
     }
 }
 
+exports.updateUserGarmentTags = async (userId, garmentId, requestBody) => {
+    let foundUser = await User.findById(userId);
+    let index = null;
+
+    for(let i = 0; i < foundUser.garments.length; i++) {
+        if(foundUser.garments[i].garment_id == garmentId) {
+            index = i;
+            break;
+        }
+    }
+
+    foundUser.garments[index].tags = requestBody.tags;
+    foundUser = await foundUser.save();
+
+    return {
+        success: true,
+        data: foundUser
+    }
+}
+
+exports.deleteUserGarment = async (userId, garmentId) => {
```

Comment:
This controller method was already implemented in kylemart/refactored-api. Why was it re-implemented here? Did you not merge it in earlier? https://github.com/Outfittr/outfittr-api/blob/edd23dd927ef27f41ce82cdfec40c813881813ab/controllers/users.controller.js#L50

Reply:
Would you consider using the syntactically-concise way of querying sub-documents that was originally written in my prior commit? If not, I honestly don't mind at this point lol.